cos-fleetshard-it/cucumber/src/main/java/org/bf2/cos/fleetshard/it/cucumber/ConnectorSteps.java

```diff
@@ -279,7 +280,19 @@ public void deploy_connector() {
 
     @When("^set configmap to:")
     public void change_configmap(Map<String, String> contents) {
-        getConfigMapFilter().accept(configMap -> configMap.setData(contents));
+        var sb = new StringBuilder();
+        sb.append("|-");
+        contents.forEach((k, v) -> {
+            sb.append("\t");
```

Comment:
not sure if this is needed either, not sure if properties trims this out when reading back

Reply:
ah right